examples/cg.py

```diff
@@ -271,8 +276,42 @@ def run_cg(
         dest="conv_threshold",
         help="convergence check threshold",
     )
+    parser.add_argument(
+        "--package",
+        dest="package",
+        type=str,
+        default="legate",
+        help="NumPy package to use (legate, numpy, or cupy)",
+    )
+    parser.add_argument(
+        "--cupy-allocator",
+        dest="cupy_allocator",
+        type=int,
+        default=0,
+        help="cupy allocator to use (0-default memory pool, 1-turn off memory pool, 2-managed memory pool)",
```

Comment:
This line, and some other ones, is too long, and pre-commit does not know how to fix it so the CI fails.

Reply:
Fixed by using long string with """.